fiaas_skipper/__init__.py

```diff
@@ -74,11 +76,18 @@ def main():
                 log.exception("Unable to load spec config extension file {!r} using defaults"
                               .format(cfg.spec_file_override))
         if cfg.enable_crd_support:
+            status_tracker = StatusTracker(cluster=cluster, application=FiaasApplication)
             deployer = CrdDeployer(cluster=cluster, release_channel_factory=release_channel_factory,
-                                   bootstrap=CrdBootstrapper(), spec_config_extension=spec_config_extension)
+                                   bootstrap=CrdBootstrapper(), spec_config_extension=spec_config_extension,
+                                   status=status_tracker)
         elif cfg.enable_tpr_support:
+            status_tracker = StatusTracker(cluster=cluster, application=PaasbetaApplication)
             deployer = TprDeployer(cluster=cluster, release_channel_factory=release_channel_factory,
-                                   bootstrap=TprBootstrapper(), spec_config_extension=spec_config_extension)
+                                   bootstrap=TprBootstrapper(), spec_config_extension=spec_config_extension,
+                                   status=status_tracker)
+        # Do period checking of deployment status across all namespaces
+        status_tracker.daemon = True
```

Comment:
Why not set the daemon flag in the constructor of the class?

Reply:
Done